src/main/java/net/minecraftforge/event/RegistryEvent.java

```diff
@@ -77,6 +77,21 @@ public Register(ResourceLocation name, IForgeRegistry<T> registry)
             return registry;
         }
 
+        /** @see IForgeRegistry#register(IForgeRegistryEntry) */
+        public void register(final T value) {
```

Comment:
geez braces! same below.

Reply:
I have no idea what you're talking about! :innocent: